pythonforandroid/toolchain.py

```diff
@@ -78,7 +78,7 @@ def __getattr__(self, key):
 sys.path.insert(0, join(toolchain_dir, "tools", "external"))
 
 
-DEFAULT_ANDROID_API = 15
+DEFAULT_ANDROID_API = 16
```

Comment:
I think 15 was chosen as the default because it was properly supported (the old option, 14, wasn't) - is this not the case any more? Why choose 16?